src/pages/guidelines/2x-grid/usage.mdx

```diff
@@ -0,0 +1,682 @@
+---
+title: 2x Grid
+description:
+  The grid is the framework for all the visual elements and typography at IBM.
+  It’s fundamental to everything we design.
+tabs: ['Overview', 'Usage', 'Code']
+---
+
+import { IbmSecurity } from '@carbon/icons-react';
+import { gridTable } from './Grid.module.scss';
+
+<PageDescription>
+
+You’ve learned the grid fundamentals, now let’s put them togther. A good layout
+requires careful planning and use of composition, which relies on principles
+such as hierarchy, scale, proportion, contrast, harmony, rhythm, repetition, and
+many more.
+
+</PageDescription>
+
+<InlineNotification>
+
+Note: Although Carbon is open source, much of our Pattern Asset Library (PAL)
+content is gated—so many of the links below can only be accessed by IBM
+employees.
```

Comment:
This is a bummer. I really want to put together some layout samples live with code that we can share publicly.